vlog4j-core/src/main/java/org/semanticweb/vlog4j/core/reasoner/implementation/InMemoryDataSource.java

```diff
@@ -101,4 +101,10 @@ public String toConfigString() {
 		return null;
 	}
 
+	@Override
+	public String getSyntacticRepresentation() {
+		// TODO Auto-generated method stub
```

Comment:
This method should not return null.
@mkroetzsch what should we return?
We don't want to display data if it is too big.

Reply:
Should return the serialized content of this DataModel, which starts with a comment "serialising", and have one fact per line.